include/GLFW/glfw3.h

```diff
@@ -2276,6 +2324,10 @@ GLFWAPI void glfwGetWindowFrameSize(GLFWwindow* window, int* left, int* top, int
  *  @errors Possible errors include @ref GLFW_NOT_INITIALIZED and @ref
  *  GLFW_PLATFORM_ERROR.
  *
+ *  @remark @wayland There is no concept of iconification in wl_shell, this
+ *  function will always emit @ref GLFW_PLATFORM_ERROR.  TODO: move to
```

Comment:
True, this was more of a reminder to myself about implementing that, I guess I’ll open an issue for every remark I had to add to the documentation that was caused by a missing GLFW feature.